scripts/shell/setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+# -ET: propagate DEBUG/RETURN/ERR traps to functions and subshells
+# -e: exit on unhandled error
+# pipefail: any failure in a pipe causes the pipe to fail
+set -eET -o pipefail
+[[ -n "${DEBUG:-}" ]] && set -x
+
+LOG_SUCCESS='\033[0;32m'
+LOG_WARNING='\033[1;33m'
+LOG_EMPHASIS='\033[0;36m'
+NO_COLOR='\033[0m'
+
+echo_with_color()
+{
+    echo -e "${2}${1}${NO_COLOR}"
+}
+
+echo ""
+echo "Welcome to the SpatialOS GDK for Unity FPS Starter Project setup script." 
+echo ""
+
+PS3="Please make a selection: "
+echo "Select a cloning method."
+echo ""
+options=("HTTPS" "SSH" "Quit")
+select opt in "${options[@]}"
+do
+    case $opt in
+        "HTTPS")
+            CLONE_URI="https://github.com/spatialos/gdk-for-unity.git"
+            break
+            ;;
```

Comment:
What a well-designed "language"

Reply:
If only we could write these in a sane scripting language

Reply:
request changes: pls use javascript

Reply:
I said **a sane scripting language** 😡